front/src/components/member/MemberUpdateModal.vue

```diff
@@ -6,7 +6,10 @@
         <VTextField
           label="새 닉네임을 입력하세요."
           color="grey darken-2"
+          maxlength="15"
+          :counter="15"
           v-model="newNickname"
+          :rules="[rules.violated, rules.unchanged]"
```

Comment:
```suggestion
          :rules="Object.values(rules)"
```
기존의 방식은 룰이 추가될 때마다 같이 수정해줘야 할거예요.
대신 이렇게 표현할 수도 있겠네요 😃